build.gradle

```diff
@@ -19,6 +19,7 @@ dependencies {
     compile group: 'org.seleniumhq.selenium', name: 'selenium-java', version: '3.4.0'
     testCompile group: 'info.cukes', name: 'cucumber-testng', version: '1.2.5'
     testCompile group: 'org.testng', name: 'testng', version: '6.11'
```

Comment:
For now we are not using it, in the future user stories we will use it. Likewise I delete it ?